website/pages/index.html

```diff
@@ -729,6 +736,8 @@ <h4 class="display-3">See it in action</h4>
     <!-- Required theme scripts (Do not remove) -->
     <script type="text/javascript" src="assets/js/theme.js"></script>
     <!-- Removes page load animation when window is finished loading -->
+    <script type="text/javascript" src="../js/mrare/contributors.js"></script>
```

Comment:
@lepasq A clarification: Is the `contributor.js` file getting babel transpired? 

Just asking as you have used `let` and `const` in the code.

Reply:
@manojVivek To be honest, I hadn't thought about this. Currently, `contributor.js` is not getting babel transpiled.  Do you want me to change all `let` and `const` keywords to `var`?

Reply:
Yes, please.

Reply:
I think `let` and `const` are valid now in plain js btw

Reply:
Yeah, just in case the user is browsing in a really old version of the browser. 😅